recipes/static/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "static" %}
+{% set version = "1.1.1" %}
+{% set md5 = "75c738ed25cc69a71299b93369ee4e40" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    preserve_egg_dir: True
```

Comment:
This is a little unusual. I think it would be healthy to comment on the reason for that (lightly commented). If this wasn't set, would the tests fail?